chunky/src/res/se/llbit/chunky/ui/SceneChooser.fxml

```diff
@@ -9,28 +26,28 @@
 <?import javafx.scene.layout.VBox?>
 
 
-<VBox maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefWidth="600.0" spacing="10.0" xmlns="http://javafx.com/javafx/8.0.40" xmlns:fx="http://javafx.com/fxml/1" fx:controller="se.llbit.chunky.ui.SceneChooserController">
-   <children>
-      <Label text="Select scene to load:" />
-      <TableView fx:id="sceneTbl" maxHeight="1.7976931348623157E308" prefHeight="293.0" VBox.vgrow="ALWAYS">
-        <columns>
-          <TableColumn fx:id="nameCol" prefWidth="120.0" text="Name" />
-          <TableColumn fx:id="chunkCountCol" prefWidth="88.0" text="Chunks" />
-            <TableColumn fx:id="sizeCol" prefWidth="104.0" text="Size" />
-            <TableColumn fx:id="sppCol" prefWidth="134.0" text="Current SPP" />
-            <TableColumn fx:id="renderTimeCol" prefWidth="133.0" text="Render time" />
-        </columns>
-      </TableView>
-      <HBox alignment="TOP_RIGHT" spacing="10.0">
-         <children>
-            <Button fx:id="deleteBtn" mnemonicParsing="false" text="Delete" />
-            <Button fx:id="exportBtn" mnemonicParsing="false" text="Export" />
-            <Button fx:id="cancelBtn" cancelButton="true" mnemonicParsing="false" text="Cancel" />
-            <Button fx:id="loadSceneBtn" defaultButton="true" mnemonicParsing="false" text="Load selected scene" />
-         </children>
-      </HBox>
-   </children>
-   <padding>
-      <Insets bottom="10.0" left="10.0" right="10.0" top="10.0" />
-   </padding>
+<VBox maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefWidth="690.0" spacing="10.0" xmlns="http://javafx.com/javafx/8.0.40" xmlns:fx="http://javafx.com/fxml/1" fx:controller="se.llbit.chunky.ui.SceneChooserController">
+    <children>
+        <Label text="Select scene to load:"/>
+        <TableView fx:id="sceneTbl" maxHeight="1.7976931348623157E308" prefHeight="293.0" VBox.vgrow="ALWAYS">
+            <columns>
+                <TableColumn fx:id="nameCol" prefWidth="200.0" text="Name"/>
+                <TableColumn fx:id="chunkCountCol" prefWidth="100.0" text="Chunks"/>
+                <TableColumn fx:id="sizeCol" prefWidth="150.0" text="Size"/>
+                <TableColumn fx:id="sppCol" prefWidth="100.0" text="Current SPP"/>
+                <TableColumn fx:id="renderTimeCol" prefWidth="100.0" text="Render time"/>
+            </columns>
```

Comment:
It would be cool to have "last modified" as a column here so that the user can manually sort by that, too.

Reply:
It would.... but that would make the window a lot wider or have a horizontal scroll bar, and I was trying to make it too too much wider, since the window is already pretty wide. (I already increased it from 600px wide to nearly 700, and with last modified, that'd probably be like 900 wide to show the whole thing.) (If they wanted to sort by "last modified" after sorting by something else, they could close the window and open it again, though a column would make it more obvious.)

If you still think it's worth it, I can go test that out and see what it looks like.